README.md

```diff
@@ -16,8 +16,26 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
 ## Table of Contents
 
 * [About jQuery](#about-jquery)
-
+* [Avoid using the `.css()` method](#avoid-using-the-.css()-method)
 
 ## About jQuery
 
 [jQuery](http://jquery.com/) is a utility library for easy DOM access & manipulation, event handling, Ajax and more. By using jQuery you can write consise and expressive code which works across modern and legacy browsers. jQuery has extensive tests, detailed documentation, a large active community and an ecosystem of plugins.
+
+## Avoid using the `.css()` method
+### Why?
+In most cases toggling a class can accomplish the same thing.
+By not using the  `.css()` method this can be omitted when creating a custom jQuery build to reduce the size.
+
+### How?
+``` javascript
+var $myElement = $('[my-element]');
```

Comment:
I would leave out this `var` statement. `$element.css()` etc will do as another rule explains caching jQuery objects and prefixing them with a `$`.